### ISSUE TITLE
Fonts without a "ẞ" are still displayed with "German" language filter

### ISSUE THREAD

**Issue by thlinard** [google/fonts#8142]
**Describe the issue**
According to https://github.com/googlefonts/lang/blob/main/Lib/gflanguages/data/languages/de_Latn.textproto ([#153](https://github.com/googlefonts/lang/pull/153)), ẞ is a required character and glyph for German support.

However, with the new language filter system on the website, if you select "German", you get fonts that do not have a ẞ.

**To Reproduce**
Select the language filter "German" and put an ẞ in Preview, like this: https://fonts.google.com/?preview.text=%E1%BA%9E&lang=de_Latn

**Comment by emmamarichal** [comment]
I still see `not_required: "ẞ"` in the .textproto.

But indeed, this character is now considered as the official one (instead of `SS`).
There were discussions about this a few weeks ago, @davelab6, what were the conclusions? Should we work on updating this filter, so that fonts that don't have this character are removed from the list?

**Comment by thlinard** [comment]
> I still see `not_required: "ẞ"` in the .textproto.

Oh, my mistake. "ẞ" is both in "base" (what misled me) and in "not_required".

But indeed the conclusion of ([#153](https://github.com/googlefonts/lang/pull/153)) was to make it a required character.

**Comment by emmamarichal** [comment]
Yes, I don't know yet for the old fonts, but for the new ones, this character is definitely required :)

**Comment by thlinard** [comment]
Dave was more specific in ([#152](https://github.com/googlefonts/lang/pull/152))

> Alright, I am convinced that, if a user searches for fonts that support German, and then Roboto, Poppins, Lato etc **should not** come up.

**Comment by emmamarichal** [comment]
@davelab6 @chrissimpkins I let you plan/assign this work!

**Comment by chrissimpkins** [comment]
Discussed with Nate today. The filter fix is in progress.
